README.md

```diff
@@ -72,6 +72,8 @@ The following custom properties and mixins are available for styling:
 | --- | --- | --- |
 | `--paper-icon-button-disabled-text` | The color of the disabled button | `--disabled-text-color` |
 | `--paper-icon-button-ink-color` | Selected/focus ripple color | `--primary-text-color` |
+| `--iron-icon-width` | Width of iron-icon | `100%` |
```

Comment:
These are not needed -- they're properties that are exposed by `iron-icon`: https://github.com/PolymerElements/iron-icon/blob/master/iron-icon.html#L72

I would actually just add a line somewhere before this section, along the lines of: "To style the internal icon, please see the`Polymer.IronIcon` documentation"